packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1480,6 +1480,9 @@ abstract class PlatformViewController {
   /// an initial position.
   Future<void> create({Size? size, Offset? position}) async {}
 
+  /// Moves the platform view to a new [FlutterView].
+  Future<void> moveToFlutterView(int flutterViewId) async {}
```

Comment:
Could we update the `PlatformViewController.viewId` documentation on lines 1451-1457 to clarify that identifier is for the platform view (instead of a Flutter view)?